Source/ACE.Database/ICharacterDatabase.cs

```diff
@@ -20,7 +20,7 @@ public interface ICharacterDatabase
         
         Task<bool> CreateCharacter(Character character);
 
-        Task UpdateCharacter(Character character);
+        void UpdateCharacter(Character character);
```

Comment:
This is required to be a void due to changes in lines 167 of the CharacterDatabase.cs file.